### ISSUE TITLE
[1.0.13] Asus Nexus 7 / Android 5.1.1 - My Video - Application crashes when user taps physical back button on Find Music screen and then on My Video screen

### ISSUE THREAD

**Issue by vysohanych** [Lomotif/lomotif-android-issues#25]
Frequency: Every Time

Action:
1. Launch Lomotif
2. Tap START button
3. Select any video on My Video screen
4. Tap apply button on My Video screen
5. Tap physical back button on Find Music screen
6. Tap physical back button on My Video screen

Expected result: User sees home screen

Actual result: Application crashes

Video: [Recording.zip](https://github.com/Lomotif/lomotif-android-issues/files/178860/Recording.zip)

**Comment by jpaulo1205** [comment]
Note: Fixed for next build (1.0.14)

**Comment by hiteshbheda** [comment]
@xiuie @jpaulo1205 I can replicate it every time on Sony xPeria Z / Android 5.1.1

**Comment by hiteshbheda** [comment]
@xiuie @jpaulo1205 Yes, i can still reproduce it EVERY TIME in build [1.0.14] on Sony xPeria Z / Android 5.1.1

**Comment by jpaulo1205** [comment]
@hiteshbheda @vysohanych can you still replicate this issue?

**Comment by hiteshbheda** [comment]
@jpaulo1205 @xiuie I can't reproduce this bug on build [1.0.15] on Sony xPeria Z / Android 5.1.1

I have tried to reproduce this multiple times but no crashing. You can close it.

**Comment by vysohanych** [comment]
I can't reproduce it on build 1.0.18 on Asus Nexus 7 Android 5.1.1

**Comment by hiteshbheda** [comment]
@xiuie @jpaulo1205 I can't reproduce this issue for android app build [1.0.18] on Asus Zenfone Go 4.5 / Android 5.1

**Comment by vysohanych** [comment]
I can't reproduce it on build 1.0.18 on Asus Nexus 7 Android 5.1.1.

**Comment by hiteshbheda** [comment]
@xiuie @jpaulo1205 I can't reproduce this issue for build [1.0.19.1] on Asus Zenfone Go 4.5 / Android 5.1
